kernelci/rootfs.py

```diff
@@ -52,16 +53,27 @@ def _build_debos(name, config, data_path, arch):
     return shell_cmd(cmd, True)
 
 
+def _build_buildroot(name, config, data_path, arch, frag='baseline'):
+    cmd = 'cd {data_path} && ./configs/frags/build {arch} {frag}'.format(
+        data_path=data_path,
+        arch=arch,
+        frag=frag
+    )
+    return shell_cmd(cmd, True)
+
+
 def build(name, config, data_path, arch):
     """Build rootfs images.
 
     *name* is the rootfs config
     *config* contains rootfs-configs.yaml entries
-    *data_path* points to debos location
+    *data_path* points to debos or buildroot location
     *arch* required architecture
     """
     if config.rootfs_type == "debos":
         return _build_debos(name, config, data_path, arch)
+    elif config.rootfs_type == "buildroot":
```

Comment:
This is OK for now, to get the functionality in place.  As a follow-up it would be nice to start using classes with a factory based on the config name, rather than `if type is ...` logic in several places.  I can already see 3 object or class methods: `build`, `validate`, `dump_config`.

Reply:
Good point, I make take care of it in next iteration